packages/mobile/src/fiatExchanges/reducer.ts

```diff
@@ -33,6 +34,12 @@ export const providersDisplayInfo: { [provider in CicoProviderNames]: ProviderFe
     icon:
       'https://firebasestorage.googleapis.com/v0/b/celo-mobile-mainnet.appspot.com/o/images%2Ftransak.png?alt=media',
   },
+  [CicoProviderNames.Xanpool]: {
+    name: 'Xanpool',
+    icon:
+      // TODO: update image
+      'https://firebasestorage.googleapis.com/v0/b/celo-mobile-mainnet.appspot.com/o/images%2Ftransak.png?alt=media',
```

Comment:
Reminder to do this :P

Reply:
Thanks - waiting on their team to provide us with a logo!